development/buildExamples.ts

```diff
@@ -38,8 +55,12 @@ async function buildExamples() {
         try {
           await build({
             src: srcPath,
-            dist: path.resolve(exampleFilePath, 'dist'),
+            dist: resolve(exampleFilePath, 'dist'),
             sourceMaps: true,
+            stripComments: true,
+            port: 8000,
+            _: [''],
+            $0: '',
```

Comment:
Oh I see, yargs expects those properties to be present. Hm. We can create an interface accepted by the `build` command handler that doesn't include those properties. We'll thank ourselves later when we write the tests. Feel free to go ahead with that, otherwise I'll tinker with it later.